charts/incubator/alfresco-dbp/values.yaml

```diff
@@ -29,15 +177,17 @@ alfresco-content-services:
     activemq:
       enabled: false
   repository:
-# DEPLOY-466 / DEPLOY-467 - Remove image pin as we are using the ACS provided images
-#    image:
-#      repository: quay.io/alfresco/alfresco-dbp-repository
-#      tag: "0.3-SNAPSHOT"
+  # DEPLOY-466 / DEPLOY-467 - Remove image pin as we are using the ACS provided images
+  #    image:
+  #      repository: quay.io/alfresco/alfresco-dbp-repository
+  #      tag: "0.3-SNAPSHOT"
```

Comment:
Since DEPLOY-466 and DEPLOY-467 happened some time ago and we don't anticipate the need to override ACS image parameters, how about tidying this up?

Reply:
true, its annoying, we have an issue. its enough.